bot/bot.py

```diff
@@ -72,6 +80,14 @@ async def close(self) -> None:
         if self.redis_session:
             await self.redis_session.close()
 
+    async def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
+        """Load the extension only if it is not present in unloaded cache."""
+        if await self.unloads_cache.get(name):
+            self.successfully_unloaded.append(name)
+            log.info(f"Skipping cog {name}, found in unloaded cache.")
+        else:
+            super().load_extension(name, package=package)
```

Comment:
Shouldn't you also override `unload_extension()` to add the extension to the unloaded cache?